ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -1,12 +1,11 @@
 [
   {
-    "id": "java-theia-docker",
+    "id": "simple-theia-docker",
     "creator": "ide",
-    "name": "Java Theia (docker)",
-    "description": "Che development + Theia in a sidecar container (docker)",
+    "name": "Simple Theia (docker)",
```

Comment:
I would prefer to remove brackets, because stack validator says that the field is invalid (painful on copying the stack).
What about `Theia on docker`?

Reply:
Agree with you, maybe Theia IDE on docker?